src/components/internals/calendar-day/calendar-day.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import omit from 'lodash.omit';
+import classname from 'classnames';
+import styles from './calendar-day.mod.css';
+
+const DateCalendarDay = props => (
+  <li className={styles.wrapper}>
+    <div
+      className={classname(
+        do {
+          if (props.type === 'heading') styles.heading;
+          else if (props.type === 'spacing') styles.spacing;
+          else styles.day;
```

Comment:
You're doing this so only one class can be applied, correct? Using the object `classnames` notion would yield something else?

Reply:
Exactly, with `classnames` we get multiple things applied. I found this easier to read.